java/BUILD.bazel

```diff
@@ -155,6 +155,14 @@ genrule(
     """,
 )
 
+genrule(
+    name = "redis-server",
+    srcs = ["//:redis-server"],
+    outs = ["src/main/resources/redis-server"],
+    cmd = "cp $< $@",
+    output_to_bindir = 1,
+)
```

Comment:
Why is it not possible to just use the binary directly instead of copying it?

Reply:
This is needed by Maven. Maven doesn't know the path of the binaries generated from Bazel.

Reply:
Is it not possible to tell it the path? (I assume you're telling it the current path already somehow, right?)

Reply:
Maven needs a list of binary files, in order to package them in a jar. 
As there is no programmable interface for that, currently "copying the files to a folder" is just the way to "tell maven the path".
But maybe using symbol links would be better @chaokunyang

Reply:
Oh interesting, I didn't realize. Yeah I guess see if symlinking works, but if it's painful then just copy I guess. I feel like there must be a better solution but I can't think of any.